app/controllers/provider/invitee_signups_controller.rb

```diff
@@ -49,7 +50,7 @@ def can_create?
   end
 
   def build_user
-    @user = @invitation.make_user(params[:user] || {})
+    @user = @invitation.make_user(params.require(:user) || {})
```

Comment:
This is not the same behaviour.

`params.require` raises an exception if it is not present.
What we need here is list all attributes 
```ruby
    permitted_attributes = ????? # Need to list all of the permitted ones to create a User from an invitation
    user_attributes = params.permit(user: permitted_attributes) || {}
    @user = @invitation.make_user(user_attributes)
```